### PR TITLE
Add documentation of Feature Throttle Policy Reset Support

### PR DESCRIPTION
**Pull request by Chanuka-ChandraYapa**
Related Issue: https://github.com/wso2/api-manager/issues/2455

### CODE REVIEW

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Reset Application Throttling Policies
+
+WSO2 API Manager allows you to reset the Application-level rate limiting tier for a specified end-user. This feature can be used by the application owner or shared owner to reset throttle policies by providing the username of the application user on their request. 
+
```
Comment (rmsamitha) [inline en/docs/design/rate-limiting/resetting-application-throttling-policies.md:4]: This should be "username or uuid of the user" right?

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+[![Application Overview]({{base_path}}/assets/img/learn/application-overview-reset.png)]({{base_path}}/assets/img/learn/application-overview-reset.png)
+
+For each application user, a counter is maintained at the traffic manager of API Manager. When the counter reaches the limit defined by the application throttle policy, the relevant application user will be throttled out and further API invocation will be restricted. If the user wants to continue invoking APIs using the application with same application throttle policy, the application owner can accommodate that by resetting the quota used by that specific user only.
+
+1. Fill in the **username** of the user whose quota needs to be reset and click **Next**.
+
```
Comment (rmsamitha) [inline en/docs/design/rate-limiting/resetting-application-throttling-policies.md:10]: This should be "username or uuid" right?

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    [![Application Reset Dialog Box]({{base_path}}/assets/img/learn/application-reset-dialog.png){: style="width:55%"}]({{base_path}}/assets/img/learn/application-reset-dialog.png)
+
+    !!! note
+        The username you provided will not be validated at the API Manager level. You will not get feedback on whether the intended user's quota is reset. Hence, Make sure the username is correct and valid before proceeding.
```
Comment (rmsamitha) [inline en/docs/design/rate-limiting/resetting-application-throttling-policies.md:14]: This should be "username or uuid of the user" right?

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Fill in the **username** of the user whose quota needs to be reset and click **Next**.
+
+    [![Application Reset Dialog Box]({{base_path}}/assets/img/learn/application-reset-dialog.png){: style="width:55%"}]({{base_path}}/assets/img/learn/application-reset-dialog.png)
+
+    !!! note
+        The username you provided will not be validated at the API Manager level. You will not get feedback on whether the intended user's quota is reset. Hence, Make sure the username is correct and valid before proceeding.  
+
+2. Confirm whether the username is correct and click **Reset**.  
+
```
Comment (rmsamitha) [inline en/docs/design/rate-limiting/resetting-application-throttling-policies.md:17]: This should be "username or uuid of the user" right?

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md (hunk 2 above)
Comment (rmsamitha) [reply en/docs/design/rate-limiting/resetting-application-throttling-policies.md:10]: Check at all the places you had mentioned "username". I have pointed only few places

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+Upon clicking the **Reset** button, a reset request will be sent to the traffic manager with the username and a notification saying 'Reset request has been triggered successfully' will pop up. This does not guarantee that the reset is complete. In case of invalid or incorrect usernames, the notification may still pop up. User may have to invoke the APIs using the application to verify reset is successful.
+
+!!! info
+    **How to find the username ?**
+
+    The username used by the traffic manager to recognize the user will not be straightforward to find in some cases. 
+
+    -   If the user uses Oauth grant types such as [**Password Grant**]({{base_path}}/design/api-security/oauth2/grant-types/password-grant/), [**Authorization Code Grant**]({{base_path}}/design/api-security/oauth2/grant-types/authorization-code-grant/) to obtain the access token, username will be the **uuid** of the user. 
+    -   For [**Client-Credentials Grant**]({{base_path}}/design/api-security/oauth2/grant-types/client-credentials-grant/) type, it will be the normal username of the application owner.(Since you are accessing apis on behalf of the client and individual user is not direclty involved).
+
```
Comment (rmsamitha) [inline en/docs/design/rate-limiting/resetting-application-throttling-policies.md:29]: Interchange the order of above two points.

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md (hunk 2 above)
Comment (Chanuka-ChandraYapa) [reply en/docs/design/rate-limiting/resetting-application-throttling-policies.md:10]: Fixed with https://github.com/wso2/docs-apim/pull/7990/commits/880dbfccec85687455ced5e2fa4c751e1a5333d7

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md (hunk 3 above)
Comment (Chanuka-ChandraYapa) [reply en/docs/design/rate-limiting/resetting-application-throttling-policies.md]: Fixed with https://github.com/wso2/docs-apim/pull/7990/commits/880dbfccec85687455ced5e2fa4c751e1a5333d7

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md (hunk 4 above)
Comment (Chanuka-ChandraYapa) [reply en/docs/design/rate-limiting/resetting-application-throttling-policies.md:17]: Fixed with https://github.com/wso2/docs-apim/pull/7990/commits/880dbfccec85687455ced5e2fa4c751e1a5333d7

File: en/docs/design/rate-limiting/resetting-application-throttling-policies.md (hunk 5 above)
Comment (Chanuka-ChandraYapa) [reply en/docs/design/rate-limiting/resetting-application-throttling-policies.md:29]: Fixed with https://github.com/wso2/docs-apim/pull/7990/commits/880dbfccec85687455ced5e2fa4c751e1a5333d7

**Review by rmsamitha — APPROVED** [review]

**Pull request merged**